Source/Core/Core/ConfigManager.cpp

```diff
@@ -214,6 +214,7 @@ void SConfig::SaveCoreSettings(IniFile& ini)
   core->Set("SyncGpuOverclock", fSyncGpuOverclock);
   core->Set("FPRF", bFPRF);
   core->Set("AccurateNaNs", bAccurateNaNs);
+  core->Set("DisableICache", bDisableICache);
```

Comment:
I don't think we need to save this value to the global config.

Reply:
I guess I'm a bit confused as to how the global config works.  Are there other options that are read from it (or game-specific configs only) but not saved?

Reply:
Looks like `BBDumpPort`, `FastDiscSpeed`, and `LowDCBZHack` are also read but not saved; at least `LowDCBZHack` seems equivalent so I'll remove the save.